tools/windows/resource.rc.in

```diff
@@ -0,0 +1,32 @@
+#include "winresrc.h"
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION @VERSION_MAJOR@,@VERSION_MINOR@,@VERSION_PATCH@
+ PRODUCTVERSION @VERSION_MAJOR@,@VERSION_MINOR@,@VERSION_PATCH@
+ FILEFLAGSMASK VS_FFI_FILEFLAGSMASK
+#ifdef DEBUG
+ FILEFLAGS 1
+#else
+ FILEFLAGS 0
+#endif
+ FILEOS VOS_NT
+ FILETYPE VFT_DLL
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "\0"
+            VALUE "FileDescription", "@TARGET_NAME@ @VERSION_STRING@\0"
+            VALUE "InternalName", "@TARGET_NAME@-@VERSION_MAJOR@.dll\0"
+            VALUE "LegalCopyright", "Copyright (C) 2007-2018 Laurent Gomila\0"
```

Comment:
Is there a way to automatically have the current year added?
Otherwise, please update to 2021 😉